tests/requirements.txt

```diff
@@ -1,5 +1,3 @@
-black
-isort
```

Comment:
How about adding a `.vscode/extensions.json` file to suggest that people install the `ms-python.black-formatter` and `ms-python.isort` extensions?

```json
{
    // See https://go.microsoft.com/fwlink/?LinkId=827846
    // for the documentation about the extensions.json format
    "recommendations": [
        "ms-python.python",
        "ms-python.vscode-pylance",
        "ms-python.black-formatter",
        "ms-python.isort",
    ]
}
```